packages/input-edit/src/dev/textResources.json

```diff
@@ -1,2 +1,3 @@
 [
+  "client.component.pagination.text"
```

Comment:
textresources of "components" can be added here: 
packages/tocco-util/src/mockData/data/textResources.json